src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -142,7 +142,7 @@ extern "C" HRESULT CallApplicationMain(PCALL_APPLICATION_MAIN_DATA data)
     if (!coreClr)
     {
         char* error = dlerror();
-        fprintf(stderr, "failed to locate coreclr.dll with error %s\n", error);
+        fprintf(stderr, "failed to locate coreclr.so with error %s\n", error);
```

Comment:
Unrelated to your change, but we should not hardcode .so since it will be a .dylib on Darwin.  This is also hardcoded in `LoadCoreClr`